src/util/object_factory_parameters.cpp

```diff
@@ -46,6 +47,24 @@ void object_factory_parameterst::set(const optionst &options)
     min_nondet_string_length =
       options.get_unsigned_int_option("min-nondet-string-length");
   }
+  if(options.is_set("java-assume-input-limits"))
+  {
+    const auto &limits = options.get_option("java-assume-input-limits");
+    const auto &colon = limits.find(':');
+    DATA_INVARIANT(
```

Comment:
:no_entry_sign: User errors should throw an exception rather than an invariant. Specifically `user_input_error_exceptiont`

Reply:
Done, I used `invalid_command_line_argument_exceptiont`